sdk/netapp/arm-netapp/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/arm-netapp",
   "author": "Microsoft Corporation",
   "description": "AzureNetAppFilesManagementClient Library with typescript type definitions for node.js and browser.",
-  "version": "9.0.0",
+  "version": "10.0.0",
```

Comment:
Are we sure that the changes in this PR are breaking changes and therefore this needs a major version update?

Reply:
Hi @ramya-rao-a , I bump version since a data type name of volume is changed, should this be thought as breaking?
https://github.com/Azure/azure-sdk-for-js/pull/9198/files#diff-22503809c9acc5b5aa05ca0e60ef9ea9L1009

Reply:
ok, that makes sense. I saw that `MountTarget` still exists and so thought that adding of `MountTargetProperties` was an additional change and not breaking.

Reply:
@changlong-liu  is this breaking change caused their change of x-ms-client-flatten in swagger?

Reply:
Hi @qiaozha , no, this just comes from a rename in swagger